apps/cli/package.json

```diff
@@ -90,7 +97,11 @@
     "scripts": {
         "build": "run-s clean codegen compile copy-files",
         "clean": "rimraf dist",
-        "codegen": "run-p codegen:wagmi",
+        "codegen": "run-p codegen:wagmi codegen:helm",
+        "codegen:helm": "run-p codegen:helm:postgres codegen:helm:redis codegen:helm:traefik",
+        "codegen:helm:postgres": "helm template postgres oci://registry-1.docker.io/bitnamicharts/postgresql --output-dir src/k8s",
+        "codegen:helm:redis": "helm template redis --set architecture=standalone --set auth.enabled=false oci://registry-1.docker.io/bitnamicharts/redis --output-dir src/k8s",
+        "codegen:helm:traefik": "helm template traefik traefik/traefik --output-dir src/k8s",
```

Comment:
Since traefik is not using `oci://` I get a repository error.

```
Error: repo traefik not found
```

We should install the repo before with `helm repo add traefik https://traefik.github.io/charts`